src/main/resources/messages/LibertyBundles.properties

```
@@ -100,7 +100,7 @@ view.gradle.config.file=view Gradle configuration file
 liberty.tool.window.display.name=Projects
 
 # Test report actions
-test.report.does.not.exist= The ({0}) test report does not exist. Run tests to generate a test report. Ensure that your test report is generating at the correct location.
+test.report.does.not.exist= The test report does not exist. Run tests to generate a test report. Ensure that your test report is generating at the correct location: {0}
```

Alternatively you could add another message and that would allow translators to translate "or" appropriately for other languages.

```suggestion
test.report.does.not.exist= The test report does not exist. Run tests to generate a test report. Ensure that your test report is generating at the correct location: {0}
test.report.does.not.exist.multiple.locations= The test report does not exist. Run tests to generate a test report. Ensure that your test report is generating at the correct location: {0} or {1}
```